Inc/DirectXMath.h

```diff
@@ -1905,8 +1956,12 @@ template<uint32_t VSLeftRotateElements, uint32_t Select0, uint32_t Select1, uint
 // separate math routine it would be reloaded.
 
 #ifndef XMGLOBALCONST
```

Comment:
What's the reason for the additional guard here? Doesn't ``attribute(weak)`` work in all cases?

Reply:
Fails on MinGW, which for same reason ahve support for `__declspec(selectany)`